server/container_create.go

```diff
@@ -904,6 +904,26 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 		return nil, err
 	}
 
+	containerMounts := specgen.Spec().Mounts
```

Comment:
move all of this in a separate function please

Reply:
fixed